batch/batch/server/globals.py

```diff
@@ -1,18 +1,37 @@
+import os
 import datetime
 import collections
 
+from .google_storage import upload_private_gs_file_from_string, download_gs_file_as_string
+from .google_storage import exists_gs_file
+
+
+if 'GOOGLE_APPLICATION_CREDENTIALS' not in os.environ:
+    os.environ['GOOGLE_APPLICATION_CREDENTIALS'] = '/gsa-key/privateKeyData'
+
+
+BATCH_BUCKET_NAME = os.environ.get('BATCH_BUCKET_NAME', 'user-rnk1e8pva3teuyfwrkbwan9eo')
```

Comment:
it feels kind of wrong to hardcode a testing default. could we update the test to set the environmental variable?